src/codegen/general.rs

```diff
@@ -322,7 +338,15 @@ pub fn define_auto_boxed_type(
 ) -> Result<()> {
     let sys_crate_name = env.main_sys_crate_name();
     writeln!(w)?;
-    writeln!(w, "glib_wrapper! {{")?;
+    writeln!(
+        w,
+        "{}::glib_wrapper! {{",
```

Comment:
As you call all macros explicitly now, make sure to also remove all `#[macro_use]` from the crates after regenerating